transmart-api-server/build.gradle

```diff
@@ -28,8 +28,9 @@ dependencies {
     compile project(':transmart-rest-api')
     compile project(':transmart-notifications')
 
-    compile "org.springframework.boot:spring-boot-starter-security"
     compile "org.springframework.security.oauth:spring-security-oauth2"
```

Comment:
still wondering what the best strategy to pick the library version would be.
I would like to investigate whether spring boot bom (AFAIK used by grails implicitly?) has a version for this library. I don't know which version it would pick if bom does not contain anything.